pkg/ksync/cli.go

```diff
@@ -11,39 +11,58 @@ import (
 	"github.com/jenkins-x/jx/pkg/util"
 )
 
-// InstallKSync install ksync
-func InstallKSync() (string, error) {
+// KSyncer contains the methods used for ksync
+type KSyncer interface {
+	GetJXBinLocation() (string, error)
+	IsInstalled(string) (string, bool, error)
```

Comment:
I think _IsInstalled_ should just return _bool_ and an error. Everything else is confusing imo.